mleap-spark-testkit/src/main/scala/org/apache/spark/ml/parity/SparkParityBase.scala

```diff
@@ -131,10 +136,38 @@ abstract class SparkParityBase extends FunSpec with BeforeAndAfterAll {
     }
   }
 
+  def checkRowWithRelTol(actualAnswer: Row, expectedAnswer: Row, relTol: Double) = {
+    require(actualAnswer.length == expectedAnswer.length,
+      s"actual answer length ${actualAnswer.length} != " +
+        s"expected answer length ${expectedAnswer.length}")
+
+    // TODO: support struct types?
+    actualAnswer.toSeq.zip(expectedAnswer.toSeq).foreach {
+      case (actual: Double, expected: Double) =>
+        assert(actual ~== expected relTol relTol)
+      case (actual: Vector, expected: Vector) =>
+        assert(actual ~= expected relTol relTol)
+      case (actual: Array[Double], expected: Array[Double]) =>
```

Comment:
Array -> Seq?

Reply:
Seq don't work.
Will match Seq[String] to case Seq[Double], I don't kown why...